### PR TITLE
adds stephanies talk to media, addresses #197

### PR DESCRIPTION
**Pull request by fboxwala**

### CODE REVIEW

File: content/W2016/W16-media-stephanie.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Title: Stephanie Morillo Talk
```
Comment (ehashman) [inline content/W2016/W16-media-stephanie.md]: Maybe 'Stephanie Morillo: "Circuit Breaker"'? Sarah's talk title was in the page title.

File: content/W2016/W16-media-stephanie.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+Date: 2016-03-22
+Category: Media
+Tags: talks
+Slug: stephanie-morillo-video-2016
+Author: Fatema Boxwala
+Summary: The amazing Stephanie Morillo gives a talk entitled Circuit Breaker
```
Comment (ehashman) [inline content/W2016/W16-media-stephanie.md]: Can you maybe say what the talk was about, and end the summary with a period?

The Sarah talk summary was "Sarah Sharp, Linux kernel developer, visits Waterloo to give a talk about getting involved with the Linux kernel and open source."

File: content/W2016/W16-media-stephanie.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+You can find more information about the event
+[here]({filename}/W2016/W16-event-stephanie-talk.md).
+
+<video width="320" height="240" controls>
+  <source src="http://mirror.csclub.uwaterloo.ca/wics/stephanie_morillo.mp4"
+          type="video/mp4">
+Your browser does not support the video tag.
+</video>
```
Comment (ehashman) [inline content/W2016/W16-media-stephanie.md:16]: The deploy looks good but the page is pretty sparse. Can we add the video download link and size to the bottom of the page? Like on the Sarah talk :)

File: content/W2016/W16-media-stephanie.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+Title: Stephanie Morillo Talk
+Date: 2016-03-22
+Category: Media
+Tags: talks
```
Comment (ehashman) [inline content/W2016/W16-media-stephanie.md]: Are there more tags we could add to this?

Comment (fboxwala) [inline content/W2016/W16-media-stephanie.md]: I couldn't see any that exist that apply, but maybe woc in tech, or technical writing? work experience

Comment (ehashman) [inline content/W2016/W16-media-stephanie.md]: From IRC:

```
08:50 <+ehashman> fatema: you can add tags
08:51 <+ehashman> I think #WoCinTech (the hashtag) would make a lot of sense 
08:51 <+ehashman> technical writing good too 
```

**Comment by fboxwala** [comment]
More reviews or mergey merge? @meltedesco @ehashman @evykassirer

**Comment by ehashman** [comment]
Few more changes! See screenshot:
screenshot at 2016-05-20 14 54 02

File: content/W2016/W16-media-stephanie.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+Title: Stephanie Morillo: “Circuit Breaker”
+Date: 2016-03-22
+Category: Media
+Tags: talks, #WocinTech, technical writing
```
Comment (ehashman) [inline content/W2016/W16-media-stephanie.md]: Capital c for `#WoCinTech`

File: content/W2016/W16-media-stephanie.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+Slug: stephanie-morillo-video-2016
+Author: Fatema Boxwala
+Summary: The amazing Stephanie Morillo gives a talk to UW students about
```
Comment (ehashman) [inline content/W2016/W16-media-stephanie.md]: This metadata needs to be on one line to be parsed correctly

**Comment by ehashman** [comment]
Modulo comments, LG for a merge.

File: content/W2016/W16-media-stephanie.md (hunk 6 above)
Comment (fboxwala) [inline content/W2016/W16-media-stephanie.md]: wow I wonder how many times I'm going to make this mistake before I get it right

**Comment by ehashman** [comment]
Please ensure you deploy the site after merging ^

I can see this isn't live.

**Comment by evykassirer** [comment]
ah okay - who has permissions to deploy the site?

and I'm pretty sure I can, but I can't find the button :(

**Comment by ehashman** [comment]
There isn't a button. The deployment instructions are in the README. You need a CSC account that's in the `wics` group.

**Comment by evykassirer** [comment]
oh okay - we should get @SrishtiG (and sys admin every term) to be in that then probably

thanks!

**Pull request merged**